src/microservices/payment/graphql/payment-mutation.resolver.ts

```diff
@@ -0,0 +1,39 @@
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+import { GqlPayment } from './models/payment-gql.model';
+import { CreatePaymentInput } from './input/create-payment.input';
```

Comment:
For files such as input, models, dto, and so on, it is good to have an index.ts file that exports all other files that are in the same directory (i.e. `input`). Thanks to that we could just do this instead
```suggestion
import { CreatePaymentInput } from './input;
```